api/src/main/java/openconsensus/metrics/DoubleCumulative.java

```diff
@@ -75,33 +75,12 @@
  * @since 0.1.0
  */
 @ThreadSafe
-public interface DoubleCumulative extends Metric {
+public interface DoubleCumulative extends Metric<DoubleCumulative.TimeSeries> {
```

Comment:
Last chance to call these `Accumulator`s instead of `Cumulative`s to avoid nouning the adjective. :D

Reply:
Some systems call this counter.

Reply:
Counters is definitely easier to explain